class/Validation.php

```diff
@@ -0,0 +1,563 @@
+<?php
+
+require_once('config/error_messages.php');
+require_once('class/DatabaseConnection.php');
+require_once('config/constants.php');
+require_once('config/database.php');
+
+/**
+* Super Class
+*
+* @author Smruti Ranjan
+*/
+class Validation
+{
+	// Data members
+	public static $form_data = array();
+	public $count;
```

Comment:
$count of errors you not using anywhere, using only to determine count > 0 
in place of you can use one flag variable like $is_error can assign TRUE/FALSE based on condition.